scripts/build_utils.sh

```diff
@@ -256,40 +256,26 @@ get_rpm_dist() {
     # creates a DISTRO_VERSION and DISTRO global variable for
     # use in constructing chacra urls for rpm distros
 
-    LSB_RELEASE=/usr/bin/lsb_release
-    [ ! -x $LSB_RELEASE ] && echo unknown && exit
+    source /etc/os-release
 
-    ID=`$LSB_RELEASE --short --id`
+    RELEASE=$VERSION_ID
+    DISTRO=$ID
 
     case $ID in
-    RedHatEnterpriseServer)
-        RELEASE=`$LSB_RELEASE --short --release | cut -d. -f1`
+    rhel)
         DIST=rhel$RELEASE
-        DISTRO=rhel
```

Comment:
We still need this function to return `DISTRO`

Reply:
I set it at line 262 directly after sourcing os-release

Reply:
Did you test on CentOS, RHEL, and Fedora and make sure it outputs the same string?

You'll need to modify it still for OpenSUSE

```
dgalloway@braggi18:~> grep ID /etc/os-release
ID="opensuse-leap"
```

Reply:
yeah, the others (rhel, centos, fedora) returns the correct lowercase form

ack on changing this, working on it now

Reply:
Seems like this is resolved now?

Reply:
> Seems like this is resolved now?

Wouldn't https://github.com/ceph/ceph-build/pull/2003/files#diff-1326d7a2d74cace6f00e1cd5d3ad15ead3fc6359eed098d2133fd72c0ec6700bR277 say `opensuse-leap` then?